### DISCUSSION TITLE
Unecessary level of precison for OKLCH colors?

### DISCUSSION THREAD

**Discussion by maybebansky** [saadeghi/daisyui#2992] [General]
I'm enjoying trying out the OKLCH color format but it seems the default colors have an unecessary level of precison. 

Eg `--p` is `49.12% .3096 275.75`. It seems exsessive for the hue channel to have 2 decimal points. Would it make a difference if it was `275.8` or `276`? Is it just like this as they were generated in some way? 

Very small point and not relaly an issue. I'm asking out of interest and because Im considering overiding the defaults with "simpler" colors to make copying and pasting between Figma easier.

**Comment by saadeghi** [comment]
The precise values are necessary because when we convert other color formats to OKLCH under the hood (for example Hex to OKLCH) it should visually give the exact same Hex value again when converted back to Hex (using a color picker or a screenshot)  
Probably it wouldn't make a difference for some colors but we won't want an inaccurate color convert function.  

Let me know if you have any questions.
